src/locales/en.json

```diff
@@ -89,7 +89,9 @@
     "unshared_text": "The owner may have revoked this sharing. You can no longer edit a note at this address.",
     "loading_error_title": "The note couldn't be loaded",
     "loading_error_text_noReturnUrl": "The note may not exists anymore, or our servers may have a temporary glitch. Please try again later.",
-    "loading_error_text_returnUrl": "The note may not exists anymore, or our servers may have a temporary glitch. Please go <a href='%{url}'>back</a> and try again later."
+    "loading_error_text_returnUrl": "The note may not exists anymore, or our servers may have a temporary glitch. Please go <a href='%{url}'>back</a> and try again later.",
```

Comment:
Is it used?

Reply:
@Crash-- yes in `editor-loading-error.jsx`

Reply:
yea I was wondering 😅